pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -373,11 +451,21 @@ def build_model(
                 var=logistic_saturation(x=channel_adstock, lam=lam),
                 dims=("date", "channel"),
             )
-            channel_contributions = pm.Deterministic(
-                name="channel_contributions",
-                var=channel_adstock_saturated * beta_channel,
-                dims=("date", "channel"),
-            )
+
+            if self.time_varying_media_effect:
+                channel_contributions = pm.Deterministic(
+                    name="channel_contributions",
+                    var=channel_adstock_saturated
+                    * beta_channel
+                    * tv_multiplier_media[:, None],
+                    dims=("date", "channel"),
+                )
+            else:
+                channel_contributions = pm.Deterministic(
+                    name="channel_contributions",
+                    var=channel_adstock_saturated * beta_channel,
+                    dims=("date", "channel"),
+                )
```

Comment:
nitpick. Think this makes it a little easier to read
```suggestion
            channel_contributions_var = channel_adstock_saturated * beta_channel
            if self.time_varying_media_effect:
                channel_contributions_var *= tv_multiplier_media[:, None]

            channel_contributions = pm.Deterministic(
                name="channel_contributions", 
                var=channel_contributions_var, 
                dims=("date", "channel"), 
            )
```

Reply:
Ya this is much nicer thanks man